index.html

```diff
@@ -12,16 +12,76 @@
     </script>
     <script src="https://ucarecdn.com/widget/2.5.5/uploadcare/uploadcare.full.min.js"></script>
     <script src="https://cdn.rawgit.com/zenorocha/clipboard.js/v1.5.12/dist/clipboard.min.js"></script>
-    <script src="vendor/aframe-master.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/socket.io/1.4.5/socket.io.min.js"></script>
+    <script src="vendor/easyrtc.js"></script>
+    <script src="https://aframe.io/releases/0.6.1/aframe.min.js"></script>
     <script src="build.js"></script>
+    <script src="https://unpkg.com/networked-aframe@^0.2.3/dist/networked-aframe.js"></script>
+    <script src="https://unpkg.com/aframe-motion-capture-components/dist/aframe-motion-capture-components.min.js"></script>
+    <script src="vendor/aframe-extras.loaders.js"></script>
     <script src="vendor/aframe-teleport-controls.min.js"></script>
     <script src="vendor/aframe-tooltip-component.min.js"></script>
+    <script src="vendor/spawn-in-circle.js"></script>
+    <script>
+      var headSchema = {
```

Comment:
We can pass these objects in directly into `addSchema` versus creating global vars